### PR TITLE
[node] fix missing zlib options parameter in v0, v4 and v6

### PR DESCRIPTION
**Pull request by marcghorayeb**
Wished I had seen that during my previous PR :)
This one adds the optional second argument which was missing in v0, v4 and v6.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes:
v6: https://nodejs.org/dist/latest-v6.x/docs/api/zlib.html#zlib_convenience_methods
v4: https://nodejs.org/docs/latest-v4.x/api/zlib.html#zlib_convenience_methods
v0: https://nodejs.org/docs/latest-v0.12.x/api/zlib.html#zlib_convenience_methods
- [x] Increase the version number in the header if appropriate. - not needed
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. - not needed

### CODE REVIEW

**Comment by marcghorayeb** [comment]
Seems like a recent linting update broke the whole node lint suite... I'll try to go through it.

**Comment by marcghorayeb** [comment]
Seeing this PR's scope is really not about linting, can we review it and still get it merged? I'll try to tackle the linting issues in another PR.

**Comment by marcghorayeb** [comment]
@wwwy3y3 any thoughts on this PR ? It seems each time I'm trying to build there are new failing tests which are not in the scope of my changes.

**Comment by marcghorayeb** [comment]
@mhegazy @andy-ms @ikokostya @hinell  Hey sorry to ping you directly - I don't know if anyone has seen this PR pass by with all the activity the repo is generating :) - do you have any input ? The build is failing but on test suites that don't have anything to do with the scope of the PR I think.

File: types/node/v7/tslint.json [hunk 1]
```diff
@@ -25,6 +25,7 @@
     "no-string-throw": false,
     "no-unnecessary-qualifier": false,
     "no-var-keyword": false,
+    "no-var": false,
```
Comment (andy-ms) [inline types/node/v7/tslint.json]: Shouldn't be necessary now that we've removed this rule?

Comment (marcghorayeb) [inline types/node/v7/tslint.json]: I've just retested against the latest master and unfortunately it seems to still be needed. I'm getting a lot of linting errors `ERROR:` 1162:9  no-var  Do not use 'var'.` if I don't have this.

Comment (andy-ms) [inline types/node/v7/tslint.json]: Try it out against travis.

Comment (marcghorayeb) [inline types/node/v7/tslint.json]: Passed! Thanks

**Comment by andy-ms** [comment]
Thanks!

**Pull request merged**